README.md

```diff
@@ -57,7 +57,7 @@ uvicorn app.main:ols --reload
 
 To send a request to the server you can use the following curl command:
 ```sh
-curl -X 'POST' 'http://127.0.0.1:8080/ols' -H2 'accept: application/json' -H 'Content-Type: application/json' -d '{"query": "write a deployment yaml for the mongodb image"}'
+curl -X 'POST' 'http://127.0.0.1:8000/ols' -H 'accept: application/json' -H 'Content-Type: application/json' -d '{"query": "write a deployment yaml for the mongodb image"}'
```

Comment:
everywhere else we explicitly run on port 8080, so please fix the start command to pass the port (uvicorn ols:app --reload --port 8080) rather than changing this spot to be out of sync w/ the rest of our config

yes there is an open debate about using 8000 vs 8080.  It is not going to matter to our users so at this point i'd prefer we just stick with 8080 (at least until we introduce TLS).